client/js/screens/SignUp/SignUp.js

```diff
@@ -1,9 +1,104 @@
-import React from "react";
-import { Text } from "react-native";
+import React, { Fragment } from "react";
+import {
+  Text,
+  ImageBackground,
+  View,
+  TextInput,
+  TouchableOpacity
+} from "react-native";
 import styles from "./styles";
+import { Form, Field } from "react-final-form";
 
-const Signup = () => {
-  return <Text>I am the Signup Screen</Text>;
+const onSubmit = async (values, createUser) => {
+  const res = await createUser({ variables: values });
+};
+
+const Signup = ({ data }) => {
```

Comment:
there is no data prop being passed from your container. Did you mean to say anything else?

Reply:
change it to signup cus that's what im passing from the container